zephyr/lvgl.c

```diff
@@ -326,8 +335,8 @@ static int lvgl_init(const struct device *dev)
 {
 	ARG_UNUSED(dev);
 
-	const struct device *display_dev =
-		device_get_binding(CONFIG_LV_Z_DISPLAY_DEV_NAME);
+	const struct device *display_dev = DEVICE_DT_GET(DISPLAY_NODE);
+
 	int err = 0;
 
 	if (display_dev == NULL) {
```

Comment:
this is no longer possible, please adjust with readiness check

Reply:
Fixed. Checking if device is ready using `device_is_ready`.